Dockerfile

```diff
@@ -140,6 +141,12 @@ RUN wget ${WGET_ARGS} https://github.com/zephyrproject-rtos/sdk-ng/releases/down
 	sh "zephyr-sdk-${ZSDK_ALT_VERSION}-linux-x86_64-setup.run" --quiet -- -d /opt/toolchains/zephyr-sdk-${ZSDK_ALT_VERSION} && \
 	rm "zephyr-sdk-${ZSDK_ALT_VERSION}-linux-x86_64-setup.run"
 
+RUN wget ${WGET_ARGS} https://static.rust-lang.org/rustup/rustup-init.sh && \
```

Comment:
please push this toward the end and after all other software is installed.

Reply:
I moved it to the end of file, but above on ENV configuration.